templates/fine-tune-llm_v2/end-to-end-examples/fine-tune-function-calling/README.ipynb

```diff
@@ -711,10 +582,9 @@
     "OPENAI_API_BASE = \"https://api.openai.com/v1\"\n",
     "\n",
     "# Base model config \n",
-    "BASE_MODEL_API_BASE = \"https://api.endpoints.anyscale.com/v1\"\n",
+    "BASE_MODEL_API_BASE = \"<Put your service URL here>\"\n",
```

Comment:
This won't work in general? The evlauation code is evaluating three models:
- GPT-4 (`OPENAI_API_BASE`)
- Finetuned model (`FINETUNED_MODEL_API_BASE`)
- Base model (`BASE_MODEL_API_BASE`)

After I serve the fine-tuned model, if I'm serving LoRA, then I can query the base model as well at the same service URL. But they might have used a full-param finetuned model and the service url would not have the base model then.

To keep things simple, we can just say that we will be doing LoRA finetuning (and then LoRA serving) throughout the example.